CMakeLists.txt

```diff
@@ -3,7 +3,10 @@ cmake_minimum_required(VERSION 3.15)
 include(CheckSymbolExists)
 include(CheckIPOSupported)
 
-project(ninja)
+project(NINJA
```

Comment:
I renamed the project to not have to deal with casing issues.

ninja_VERSION -> NINJA_VERSION